public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -112,6 +113,43 @@ abstract class DashboardScenePageStateManagerBase<T>
     return this.cache;
   }
 
+  protected async fetchHomeDashboard(): Promise<DashboardDTO | null> {
+    const rsp = await getBackendSrv().get<HomeDashboardDTO | HomeDashboardRedirectDTO>('/api/dashboards/home');
+
+    if (isRedirectResponse(rsp)) {
+      const newUrl = locationUtil.stripBaseFromUrl(rsp.redirectUri);
+      locationService.replace(newUrl);
+      return null;
+    }
+
+    // If dashboard is on v2 schema convert to v1 schema, there's curently no v2 API for home dashboard
+    if (isDashboardV2Spec(rsp.dashboard)) {
+      rsp.dashboard = transformDashboardV2SpecToV1(rsp.dashboard, {
+        name: '',
+        generation: 0,
+        resourceVersion: '0',
+        creationTimestamp: '',
+      });
```

Comment:
```suggestion
    if (isDashboardV2Spec(rsp.dashboard)) {
      rsp.dashboard = transformDashboardV2SpecToV1(rsp.dashboard, rsp.metadata);
```
Why we're not using the metadata here to generate properly a V1 dashboard with version, uid as part of the dashboard?